beacon-chain/core/electra/deposits.go

```diff
@@ -508,8 +434,7 @@ func GetValidatorFromDeposit(pubKey []byte, withdrawalCredentials []byte, amount
 		EffectiveBalance:           0,
 		PrincipalBalance:           0,
 	}
-	maxEffectiveBalance := helpers.ValidatorMaxEffectiveBalance(validator)
-	validator.EffectiveBalance = min(amount-(amount%params.BeaconConfig().EffectiveBalanceIncrement), maxEffectiveBalance)
+	validator.EffectiveBalance = min(amount-(amount%params.BeaconConfig().EffectiveBalanceIncrement), params.BeaconConfig().MaxEffectiveBalanceAlpaca)
```

Comment:
spec in the comments might also need to be updated.
`max_effective_balance` -> `MAX_EFFECTIVE_BALANCE_ALPACA`